.github/workflows/cross-package-check.yml

```
@@ -0,0 +1,31 @@
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      pull-requests: write
+    steps:
+      - uses: actions/checkout@v6
+        with:
+          fetch-depth: 0
+
+      - uses: ./.github/actions/setup
+        with:
+          build: true
```

actionlint (1.7.11)

[error] input "build" is not defined in action "Setup Seed Design" defined at "./.github/actions/setup". available inputs are "build-packages", "build-rootage"

(action)